.travis.yml

```diff
@@ -19,9 +19,7 @@ before_script:
   - python -m pydocstyle
 script:
   - python -m unittest discover
-  - if [[ "${TRAVIS_PULL_REQUEST}" != "false" ]]; then
```

Comment:
@michi88 @keep94 folks please bring back the check, is was making sure the version in setup.py is incremented as needed in every PR that's being merged